doc/CONTRIBUTING.md

```diff
@@ -337,6 +337,34 @@ for more configuration details. Everything within `python_config` is imported
 via `dictConfig` (in addition to some standard settings defined in
 `dataactcore.logging`.
 
+### Setup with Docker
+
+Install docker in your local machine by selecting your OS and hitting install from this [link](https://docs.docker.com/docker-for-mac/install).
+This [link](https://docs.docker.com/docker-for-mac/install/) will already include docker-compose.
+
+These are the files you need to add/change before you begin working in your container. `_example` or `example_` needs to be removed from these file names.
+```
+dataactcore/config_example.yml
+dataactcore/local_config_example.yml
+dataactcore/local_secrets_example.yml
+dataactvalidator/config/example_agency_codes_list.csv
+dataactvalidator/config/example_cars_tas.csv
+dataactvalidator/config/example_cgac.csv
+dataactvalidator/config/example_object_class.csv
+dataactvalidator/config/example_program_activity.csv
```

Comment:
All of these files could probably go in the general install section because that's actually something that has to be done no matter what method of installation you're using.

Reply:
If these files are already in the general section, we don't also need them here. Just refer to the earlier list.